src/Entity/SpecialSponsorship.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
Probably remove some of the getters/setters in favour of more descriptive methods.